packages/suite-web/e2e/tests/settings/t2t1-device-settings.test.ts

```diff
@@ -82,14 +82,11 @@ describe('T2T1 - Device settings', () => {
     });
 
     it('able to change homescreen in firmware >= 2.5.4', () => {
-        cy.task('startEmu', { wipe: true, model: 'T2T1', version: '2-main' });
+        cy.task('startEmu', { wipe: true, model: 'T2T1', version: '2-latest' });
```

Comment:
I am not sure why this test required `2-main`, I suspect it was an oversight.
`2-latest` IMHO makes sense for this test. :thinking: